doc/source/whatsnew/v0.21.0.txt

```diff
@@ -40,6 +40,7 @@ Other Enhancements
 - :func:`DataFrame.clip()` and :func:`Series.clip()` have gained an ``inplace`` argument. (:issue:`15388`)
 - :func:`crosstab` has gained a ``margins_name`` parameter to define the name of the row / column that will contain the totals when ``margins=True``. (:issue:`15972`)
 - :func:`Dataframe.select_dtypes` now accepts scalar values for include/exclude as well as list-like. (:issue:`16855`)
+- :func:`to_datetime` requires an unit with numerical arg (:issue:`15836`)
```

Comment:
also needs a mini-sub-section as this is a non-obvious change. show the previous and new behavior.